<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Publications - HSE Bioinformatics Lab</title>
    <link rel="stylesheet" href="css/styles.css">
</head>
<body>
    <header>
        <h1>HSE Bioinformatics Lab</h1>
        <nav>
            <ul>
                <li><a href="index.html">Home</a></li>
                <li><a href="about.html">About</a></li>
                <li><a href="news.html">News</a></li>
                <li><a href="research.html" class="active">Publications</a></li>
                <li><a href="staff.html">Staff</a></li>
                <li><a href="datasets.html">Datasets</a></li>
            </ul>
        </nav>
    </header>

    <main>
        <section class="publications">
            <h2>Our Publications</h2>
            <!-- List of publications -->
            <ul class="publication-list">
                <li>
                    <h3>Avoidance of Recognition Sites of Restriction-Modification Systems Is a Widespread but Not Universal Anti-Restriction Strategy of Prokaryotic Viruses</h3>
                    <p><strong>Authors:</strong> I. S. Rusinov, A. S. Ershova, A. S. Karyagina, et al.</p>
                    <p><strong>Journal:</strong> BMC Genomics, 2018, Vol. 19, No. 885, pp. 1-11</p>
                    <p><strong>Added:</strong> December 20, 2018</p>
                </li>
                <li>
                    <h3>Comparative Genomic Analysis of Holospora spp., Intranuclear Symbionts of Paramecia</h3>
                    <p><strong>Authors:</strong> S. Garushyants, A. Beliavskaia, D. B. Malko, et al.</p>
                    <p><strong>Journal:</strong> Frontiers in Microbiology, 2018, Vol. 9, Article 738</p>
                    <p><strong>Added:</strong> December 16, 2021</p>
                </li>
                <li>
                    <h3>Comparative Genomic Analysis of Fungal TPP-Riboswitches</h3>
                    <p><strong>Author:</strong> M. S. Gelfand</p>
                    <p><strong>Journal:</strong> Fungal Genetics and Biology, 2018, Vol. 114, pp. 34-41</p>
                    <p><strong>Added:</strong> March 17, 2019</p>
                </li>
                <li>
                    <h3>Conservation, Evolution, and Regulation of Splicing During Prefrontal Cortex Development in Humans, Chimpanzees, and Macaques</h3>
                    <p><strong>Authors:</strong> P. V. Mazin, X. Jiang, N. Fu, et al.</p>
                    <p><strong>Journal:</strong> RNA, 2018, pp. 585-596</p>
                    <p><strong>Added:</strong> March 5, 2018</p>
                </li>
                <li>
                    <h3>Cooption of Heat Shock Regulatory System for Anhydrobiosis in the Sleeping Chironomid Polypedilum vanderplanki</h3>
                    <p><strong>Authors:</strong> P. V. Mazin, M. S. Gelfand</p>
                    <p><strong>Journal:</strong> Proceedings of the National Academy of Sciences of the USA, 2018, pp. E2477-E2486</p>
                    <p><strong>Added:</strong> March 5, 2018</p>
                </li>
                <li>
                    <h3>Genome Rearrangements and Phylogeny Reconstruction in Yersinia pestis</h3>
                    <p><strong>Author:</strong> M. S. Gelfand</p>
                    <p><strong>Journal:</strong> PeerJ, 2018, Vol. 6, pp. 1-13</p>
                    <p><strong>Added:</strong> March 17, 2019</p>
                </li>
                <li>
                    <h3>Genome Rearrangements and Selection in Multi-Chromosome Bacteria Burkholderia spp.</h3>
                    <p><strong>Author:</strong> M. S. Gelfand</p>
                    <p><strong>Journal:</strong> BMC Genomics, 2018, Vol. 19, No. 1, pp. 1-17</p>
                    <p><strong>Added:</strong> March 17, 2019</p>
                </li>
                <li>
                    <h3>Machine-Learning Models for Cancer Breakpoints Prediction Based on DNA Structure Distributions</h3>
                    <p><strong>Authors:</strong> K. S. Cheloshkina, M. S. Poptsova</p>
                    <p><strong>Book Chapter:</strong> Proceedings of the 42nd Interdisciplinary School-Conference of the IPPI RAS "Information Technologies and Systems 2018", Institute for Information Transmission Problems, 2018, pp. 1-5</p>
                    <p><strong>Added:</strong> January 15, 2019</p>
                </li>
                <li>
                    <h3>Microbiomes of Gall-Inducing Copepod Crustaceans from the Corals Stylophora pistillata (Scleractinia) and Gorgonia ventalina (Alcyonacea)</h3>
                    <p><strong>Author:</strong> M. S. Gelfand</p>
                    <p><strong>Journal:</strong> Scientific Reports, 2018, Vol. 8, No. 1, pp. 1-10</p>
                    <p><strong>Added:</strong> March 17, 2019</p>
                </li>
                <li>
                    <h3>Neanderthal and Denisovan Ancestry in Papuans: A Functional Study</h3>
                    <p><strong>Author:</strong> M. S. Gelfand</p>
                    <p><strong>Journal:</strong> Journal of Bioinformatics and Computational Biology, 2018, Vol. 16, No. 2, pp. 1840011-1-12</p>
                    <p><strong>Added:</strong> March 17, 2019</p>
                </li>
                <li>
                    <h3>PQ, a New Program for Phylogeny Reconstruction</h3>
                    <p><strong>Authors:</strong> D. Penzar, M. Krivozubov, S. A. Spirin</p>
                    <p><strong>Journal:</strong> BMC Bioinformatics, 2018, Vol. 19, No. 374, pp. 1-14</p>
                    <p><strong>Added:</strong> October 26, 2018</p>
                </li>
                <li>
                    <h3>Pangenomic Definition of Prokaryotic Species and the Phylogenetic Structure of Prochlorococcus spp.</h3>
                    <p><strong>Author:</strong> M. S. Gelfand</p>
                    <p><strong>Journal:</strong> Frontiers in Microbiology, 2018, Vol. 9, pp. 1-11</p>
                    <p><strong>Added:</strong> March 17, 2019</p>
                </li>
                <li>
                    <h3>Recognition of 3' UTR Stem-Loop in LINE Transposons Across the Tree of Life by Machine Learning Methods</h3>
                    <p><strong>Authors:</strong> A. V. Shein, M. S. Poptsova</p>
                    <p><strong>Book Chapter:</strong> Proceedings of the 42nd Interdisciplinary School-Conference of the IPPI RAS "Information Technologies and Systems 2018", Institute for Information Transmission Problems, 2018, pp. 1-5</p>
                    <p><strong>Added:</strong> January 15, 2019</p>
                </li>
                <li>
                    <h3>Recognizing Patterns of Nucleosome and DNA Structures Positioning</h3>
                    <p><strong>Authors:</strong> E. A. Tevanyan, M. S. Poptsova</p>
                    <p><strong>Book Chapter:</strong> Proceedings of the 42nd Interdisciplinary School-Conference of the IPPI RAS "Information Technologies and Systems 2018", Institute for Information Transmission Problems, 2018, pp. 1-10</p>
                    <p><strong>Added:</strong> January 15, 2019</p>
                </li>
                <li>
                    <h3>Recognizing Patterns of Nucleosome and DNA Structures Positioning</h3>
                    <p><strong>Authors:</strong> E. A. Tevanyan, M. S. Poptsova</p>
                    <p><strong>Book Chapter:</strong> Proceedings of the 2018 IEEE International Conference on Bioinformatics and Biomedicine, Madrid: IEEE, 2018, pp. 2808-2809</p>
                    <p><strong>Added:</strong> February 3, 2019</p>
                </li>
                <li>
                    <h3>The Genes of the Sulphoquinovose Catabolism in Escherichia coli Are Also Associated with a Previously Unknown Pathway of Lactose Degradation</h3>
                    <p><strong>Author:</strong> M. S. Gelfand</p>
                    <p><strong>Journal:</strong> Scientific Reports, 2018, Vol. 8, No. 1, pp. 1-12</p>
                    <p><strong>Added:</strong> March 17, 2019</p>
                </li>
                <li>
                    <h3>Search for Patterns of Association Between Functional Genomic Elements</h3>
                    <p><strong>Authors:</strong> O. O. Matkarimov, D. E. Polivoda, M. S. Poptsova</p>
                    <p><strong>Book Chapter:</strong> Proceedings of the International Conference "Mathematical Biology and Bioinformatics", Vol. 7, Pushchino: Institute of Mathematical Problems of Biology RAS, 2018, Chapter 57, pp. 1-3</p>
                    <p><strong>Added:</strong> December 19, 2018</p>
                </li>
                <li>
                    <h3>Development of a Machine Learning Model Recognizing G-Quadruplexes Based on Physicochemical Properties of DNA</h3>
                    <p><strong>Authors:</strong> D. L. Konovalov, M. S. Poptsova</p>
                    <p><strong>Book Chapter:</strong> Materials of the International Youth Scientific Forum "Lomonosov-2018", Moscow: MAX Press, 2018</p>
                    <p><strong>Added:</strong> April 23, 2020</p>
                </li>
                <li>
                    <h3>Comparison of Methods for Detecting Exceptional Sequences in Prokaryotic Genomes</h3>
                    <p><strong>Authors:</strong> A. S. Ershova, I. S. Rusinov, A. S. Karyagina, et al.</p>
                    <p><strong>Journal:</strong> Biochemistry, 2018, Vol. 83, No. 2, pp. 225-237</p>
                    <p><strong>Added:</strong> October 15, 2019</p>
                </li>
                <li>
                    <h3>Chlamydia Pan-Genomic Analysis Reveals Balance Between Host Adaptation and Selective Pressure to Genome Reduction</h3>
                    <p><strong>Authors:</strong> O. Sigalova, M. S. Gelfand</p>
                    <p><strong>Journal:</strong> BMC Genomics, 2019, Vol. 20, No. 1, pp. 1-17</p>
                    <p><strong>Added:</strong> October 31, 2019</p>
                </li>
                <li>
                    <h3>Comparative Genomic Analysis of the Regulation of Aromatic Metabolism in Betaproteobacteria</h3>
                    <p><strong>Authors:</strong> I. Suvorova, M. S. Gelfand</p>
                    <p><strong>Journal:</strong> Frontiers in Microbiology, 2019, Vol. 10, pp. 1-18</p>
                    <p><strong>Added:</strong> October 31, 2019</p>
                    <p><strong>Abstract:</strong> Aromatic compounds are a common carbon and energy source for many microorganisms, some of which can even degrade toxic chloroaromatic xenobiotics. This comparative study of aromatic metabolism in 32 Betaproteobacteria species describes the links between several transcription factors (TFs) that control benzoate (BenR, BenM, BoxR, BzdR), catechol (CatR, CatM, BenM), chlorocatechol (ClcR), methylcatechol (MmlR), 2,4-dichlorophenoxyacetate ...</p>
                </li>
            </ul>
        </section>
    </main>

    <footer>
        <p>&copy; 2024 HSE Bioinformatics Lab</p>
    </footer>
</body>
</html>
